ert/ensemble_evaluator/builder/_job.py

```diff
@@ -159,8 +176,10 @@ def set_ext_job(self, ext_job: ExtJob) -> "_LegacyJobBuilder":
     def build(self) -> _LegacyJob:
         if self._id is None:
             self._id = str(uuid.uuid4())
+        if self._index is None:
+            self._index = "0"
```

Comment:
I thought you meant to make this mandatory?

Reply:
That's what the reviews are for - forgot on that one. Will change this one too!

Reply:
> Attaching index to source of the CloudEvent is an inaccuracy I think, as the spec says that "the 'source' is the context in which the occurrence happened". The job should be data on the job itself, but this is largely semantics so feel free to ignore.

There is still time to fix it. I just thought that since `index` was on the same semantic level as `id` therefore it should be in the source. But you might be right that the context should be identifiable uniquely and thus index is providing only noise into it ...

Reply:
> There is still time to fix it. I just thought that since `index` was on the same semantic level as `id` therefore it should be in the source. But you might be right that the context should be identifiable uniquely and thus index is providing only noise into it ...

I can go either way with this one, since it is down to semantics. I can create an issue and let's discuss it there?